Source/Core/DolphinWX/ISOProperties.cpp

```diff
@@ -1298,7 +1306,12 @@ void CISOProperties::OnComputeMD5Sum(wxCommandEvent& WXUNUSED (event))
 
 void CISOProperties::OnShowDefaultConfig(wxCommandEvent& WXUNUSED (event))
 {
-	LaunchExternalEditor(GameIniFileDefault);
+	for (std::string filename : SCoreStartupParameter::GetGameIniFilenames(game_id, OpenISO->GetRevision()))
+	{
+		std::string path = File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP + filename;
+		if (File::Exists(path))
+			LaunchExternalEditor(path);
```

Comment:
Maybe add a ```break``` here? Otherwise I'd assume if there are multiple files that match and exist, it would run all of them...and I'm not sure if thats intended.